### PR TITLE
Mm 03 fun fact

### PR DESCRIPTION
**Pull request by jxevergrow**
We add a component on the landing page, which display a random fact and a link to the fun fact page. On the fun fact page, three fact cards are displayed and can lead user to the subpage of corresponding categories.

### CODE REVIEW

File: src/components/App/index.tsx [hunk 1]
```diff
@@ -5,14 +5,32 @@ import { Quiz } from "../Quiz/index"; // File location finds the file "index" au
 import LandingPage from "../Layouts/LandingPage";
 import MarsRoverPage from "../Layouts/MarsRoverPage";
 
+import FunFactPage from "../Layouts/FunFactPage";
+import FunFactRoverPage from "../Layouts/FunFactRoverPage";
+import FunFactMissionPage from "../Layouts/FunFactMissionPage";
+import FunFactEnvPage from "../Layouts/FunFactEnvPage";
+
 const App: React.FC = () => {
+  // const facts: [] = factList;
```
Comment (Louisasa) [inline src/components/App/index.tsx]: can we remove this please?

File: src/components/FunFacts/FunFact.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
+$widthMobile: 430px;
```
Comment (Louisasa) [inline src/components/FunFacts/FunFact.scss]: can we use the sizes set in constants.scss

File: src/components/FunFacts/FunFact.scss [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+$fontSizeBoby: 16px;
+$fontBold: 600;
+
+.gridContainer {
```
Comment (Louisasa) [inline src/components/FunFacts/FunFact.scss]: is this used anywhere?

File: src/components/FunFacts/FunFact.scss [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  color: #dcdcdc;
+
+  .card-container {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: $widthMobile;
+
+    .fact-container {
+      background-color: #151820;
+      margin: 0 0 1rem 0;
```
Comment (Louisasa) [inline src/components/FunFacts/FunFact.scss]: can we use the margins set in constants.scss please

File: src/components/FunFacts/FunFact.scss [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+      }
+
+      .content {
+        margin-left: 1rem;
+        margin-bottom: 1rem;
+        text-align: left;
+        // font-weight: $fontBold;
+        color: whitesmoke;
+      }
+      .title {
```
Comment (Louisasa) [inline src/components/FunFacts/FunFact.scss]: can we use text set in constants.scss please

File: src/components/FunFacts/FunFact.scss [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        font-weight: $fontBold;
+        font-size: larger;
+        margin-left: 1rem;
+      }
+      .text-container {
+        margin-left: 0.5rem;
+        text-align: left;
+        // display: flex;
```
Comment (Louisasa) [inline src/components/FunFacts/FunFact.scss]: can we remove all comments please :)

File: src/components/FunFacts/FunFact.tsx [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  );
+
+  const roverRandom = roverList[Math.floor(Math.random() * 4)];
+  const missionRandom = missionList[Math.floor(Math.random() * 4)];
+  const environmentRandom = environmentList[Math.floor(Math.random() * 4)];
+
+  const funfactList = [roverRandom, missionRandom, environmentRandom];
+
+  return (
+    <body>
```
Comment (Louisasa) [inline src/components/FunFacts/FunFact.tsx]: can we use div here instead of body please

File: src/components/FunFacts/FunFactCard.tsx [hunk 8]
```diff
@@ -0,0 +1,20 @@
+import "./FunFact.scss";
+import { Fact } from "./factList";
+import { Link } from "react-router-dom";
+
+export default function FunFactCard(props: { fact: Fact; link?: string }) {
+  return (
+    <body>
```
Comment (Louisasa) [inline src/components/FunFacts/FunFactCard.tsx]: same here please

File: src/components/FunFacts/factList.ts [hunk 9]
```diff
@@ -0,0 +1,231 @@
...
+    title: "Atmosphere composition",
+    image: image12,
+    id: 12,
+    content:
+      "The Martian atmosphere is mostly carbon dioxide (CO2) . Today, Mars is a cold, inhospitable desert but features like dry riverbeds and minerals that only form with liquid water indicate that long ago it had a thick atmosphere that retained enough heat for liquid water.",
+    category: "mission",
+  },
+];
+
+// interface Fact {
```
Comment (Louisasa) [inline src/components/FunFacts/factList.ts]: can we remove these comments please

**Review by Louisasa — COMMENTED** [review]
looking good! just some little things left to tidy up :)

File: src/components/App/index.tsx (hunk 1 above)
Comment (jxevergrow) [reply src/components/App/index.tsx]: yes

**Review by Louisasa — APPROVED** [review]

**Pull request merged**